components/submit/fields/Tags.vue

```diff
@@ -46,12 +66,15 @@
     },
     methods: {
       add () {
-        if (this.query.length > 1) {
+        if (this.query.length > 1 && this.selected.length < 5 && this.selected.indexOf(this.query) === -1) {
           this.$store.dispatch('submit/addTag', this.query)
           this.$store.dispatch('submit/resetTagsResults')
           this.$mixpanel.track('New DApp - Add tag', { tag: this.query })
         }
       },
+      hasWarning (tag) {
+        return this.obviousTags.indexOf(tag) >= 0
```

Comment:
tag matching is case sensitive, I'd make it case insensitive; so both `ethereum` and `Ethereum` would give a warning